packages/bierzo-wallet/package.json

```diff
@@ -39,22 +39,25 @@
     "clipboardy": "^2.1.0"
   },
   "scripts": {
+    "use-extension-config-development": "cd ../sanes-chrome-extension && yarn use-config-development && yarn override-config-development && cd ../bierzo-wallet",
```

Comment:
I think `use-extension-config-development` should only run for e2e tests. The other tests must not touch the extension directory

Reply:
Fixing this probably fixes all the problem. Then the additional dependencies is not needed since `use-extension-config-development` is not called during the build but only when running the e2e test.